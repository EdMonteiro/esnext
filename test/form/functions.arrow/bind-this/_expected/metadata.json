{
  "functions.arrow": {
    "functions": [
      {
        "type": "FunctionExpression",
        "id": null,
        "params": [],
        "body": {
          "type": "BlockStatement",
          "directives": [],
          "body": []
        },
        "generator": false,
        "async": false,
        "returnType": null,
        "typeParameters": null
      }
    ]
  }
}