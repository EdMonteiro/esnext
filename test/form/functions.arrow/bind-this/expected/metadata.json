{
  "objects.shorthand": {
    "properties": []
  },
  "objects.concise": {
    "properties": []
  },
  "modules.commonjs": {
    "imports": [],
    "exports": [],
    "directives": []
  },
  "functions.arrow": {
    "functions": [
      {
        "type": "FunctionExpression",
        "id": null,
        "params": [],
        "body": {
          "type": "BlockStatement",
          "body": []
        },
        "generator": false,
        "expression": false
      }
    ]
  },
  "declarations.block-scope": {
    "declarations": []
  },
  "objects.destructuring": [],
  "strings.template": {
    "concatenations": []
  }
}